src/routes/apiRouter.js

```diff
@@ -283,5 +297,11 @@ router.get('/makers', makers)
 router.get('/cities/search', searchCities)
 router.get('/network/rewards', networkRewards)
 router.get('/network/rewards/averages', averageHotspotEarnings)
+router.post(
+  '/hexes/earnings',
+  cors({ origin: 'https://hotspot-tileserver.herokuapp.com/' }),
```

Comment:
@danielcolinjames @allenan I want to allow only tileserver to POST to `'api/hexes/earnings'`. I've tried messing with CORS quite a big and can't get it to work. Any ideas how to fix?

Reply:
@matthewcarlreetz hmm good question. I'm not familiar enough with POSTing stuff to endpoints to know how to get that to work unfortunately, but I think Andrew probably would know. what's the context on this again, and why a POST is needed here? I think I missed it in Discord, wondering if there's another way to achieve the same thing since explorer-api hasn't really been used this way as of yet